homework1/task04/tests/test_sum_of_four.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from homework1.task04 import sum_of_four
+
+
+@pytest.mark.parametrize(
+    "lst1, lst2, lst3, lst4, result",

```

Comment:
test against both function implementation, you can parametrize tests with _functions_ as well